app/code/Magento/CmsGraphQl/etc/schema.graphqls

```diff
@@ -13,13 +13,15 @@ type StoreConfig @doc(description: "The type contains information about a store
 type Query {
     cmsPage (
         id: Int @doc(description: "Id of the CMS page")
+        identifier: String @doc(description: "Identifier of the CMS page")
```

Comment:
You have added 'identifier' in the query but not added 'identifier' in CmsPage type. Why?
I think it would be logical to get an identifier in a return result as well.

Reply:
> You have added 'identifier' in the query but not added 'identifier' in CmsPage type. Why?

forgot 😊 

> I think it would be logical to get an identifier in a return result as well.

**fixed**